virt/managing_vms/virt-enabling-disabling-vm-delete-protection.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-enabling-disabling-vm-delete-protection"]
+= Enabling or disabling virtual machine delete protection
+include::_attributes/common-attributes.adoc[]
+:context: virt-enabling-disabling-vm-delete-protection
+
+toc::[]
+
+You can prevent the inadvertent deletion of a virtual machine (VM) by enabling delete protection. You can also disable delete protection.
+
+You enable or disable delete protection from either the {product-title} web console or the command line.
+
+.Prerequisites
+
+* You have installed {VirtProductName} 4.19 or later.
+* You have installed the OpenShift CLI (`oc`).
```

Comment:
Because the requisites are different using UI and CLI, just being common that you need 4.19, does it make sense to include in the top description?